installer/resources/pacbot_app/files/DB_Policy.sql

```diff
@@ -2546,6 +2550,18 @@ INSERT IGNORE INTO `cf_PolicyParams` (`policyID`, `paramKey`, `paramValue`, `def
 INSERT IGNORE INTO `cf_PolicyParams` (`policyID`, `paramKey`, `paramValue`, `defaultVal`, `isEdit`, `isMandatory`, `encrypt`, `displayName`, `description`) VALUES ('delete_unused_load_balancer','policyKey','delete-unused-load-balancer','','false','false','false','','');
 INSERT IGNORE INTO `cf_PolicyParams` (`policyID`, `paramKey`, `paramValue`, `defaultVal`, `isEdit`, `isMandatory`, `encrypt`, `displayName`, `description`) VALUES ('delete_unused_vm_disk','policyKey','delete-unused-vm-disk','','false','false','false','','');
 INSERT IGNORE INTO `cf_PolicyParams` (`policyID`, `paramKey`, `paramValue`, `defaultVal`, `isEdit`, `isMandatory`, `encrypt`, `displayName`, `description`) VALUES ('enable_aws_config_service','policyKey','check-aws-config-enabled','','false','false','false','','');
+INSERT IGNORE INTO `cf_PolicyParams` (`policyID`, `paramKey`, `paramValue`, `defaultVal`, `isEdit`, `isMandatory`, `encrypt`, `displayName`, `description`) VALUES ('ContrastCriticalLibraryVulnerabilityExists_version-1_Vulnerability_library','policyKey','check-vulnerability-exists-for-library','','false','false','false','','');
```

Comment:
Please lowercase the policy ids and use snake case.

Also, why do we need all the post fix _version-1_Vulnerability_library"; 1. We know it's a lib from id, 2. If you want to include a version, shorten it to v1.

Hence -> contrast_{severity}_library_vulnerability_exists_v1

Reply:
do we even need that v1? I doubt

Reply:
I'm ok with any policy ID as long as it's consistent and represents what is checked by policy.
But I ckecked other policies - and that is just hell...
Here is some example:
* delete_unused_vm_disk
* Ec2PublicAccessPortWithS5Vulnerability_version-1_Ec2PublicAccessPortWithS5Vulnerability_ec2
* core-networking-iam-user-with-unapproved-access_version-1_core-networking-iam-user-with-unapproved-access_iamuser

Since there is no versioning for policies and this is DB I would skip v1 as whole.
As for format @plyubich suggested - I would alter it a bit just to follow next logic - from big to small. So:
`{source}_{asset_type}_{vuln_name}`
Hence here we will get:
`contrast_library_{severity}_vulnerabilities_found`
source: contrast
asset_type: library
vuln_name: {severity}_vulnerabilities_found

@plyubich what do you think?

Reply:
@akashpaladin please use next pattern and apply it everywhere in your PR and naming will be consistent:
```
contrast_library_{severity}_vulnerabilities_found
```
and basic rule is
```
{source}_{asset_type}_{vuln_name}
```
Please ping me when you're done and I'll approve your PR.